sections/gcloud.zsh

```diff
@@ -0,0 +1,40 @@
+#
+# Google Cloud Platform (gcloud)
+#
+# gcloud is a tool that provides the primary command-line interface to Google Cloud Platform. 
+# Link: https://cloud.google.com/sdk/gcloud/
+
+# ------------------------------------------------------------------------------
+# Configuration
+# ------------------------------------------------------------------------------
+
+SPACESHIP_GCLOUD_SHOW="${SPACESHIP_GCLOUD_SHOW=true}"
+SPACESHIP_GCLOUD_PREFIX="${SPACESHIP_GCLOUD_PREFIX="using "}"
+SPACESHIP_GCLOUD_SUFFIX="${SPACESHIP_GCLOUD_SUFFIX="$SPACESHIP_PROMPT_DEFAULT_SUFFIX"}"
+SPACESHIP_GCLOUD_SYMBOL="${SPACESHIP_GCLOUD_SYMBOL="☁️ "}"
+SPACESHIP_GCLOUD_COLOR="${SPACESHIP_GCLOUD_COLOR="26"}"
+
+# ------------------------------------------------------------------------------
+# Section
+# ------------------------------------------------------------------------------
+
+# Shows active gcloud configuration.
+spaceship_gcloud() {
+  [[ $SPACESHIP_GCLOUD_SHOW == false ]] && return
+
+  # Check if the glcoud-cli is installed
+  spaceship::exists gcloud || return
+
+  # Check if there is an active config
+  [[ -f ~/.config/gcloud/active_config ]] || return
+ 
+  # Reads the current config from the file 
+  local GCLOUD_CONFIG=${$(head -n1 ~/.config/gcloud/active_config)}
+
```

Comment:
```suggestion
  # Get active project
  local GCLOUD_ACTIVE_PROJECT=$(grep 'project = ' .config/gcloud/configurations/config_$GCLOUD_CONFIG | cut -d ' ' -f3)
  [[ -z $GCLOUD_ACTIVE_PROJECT ]] && return
```

Reply:
This breaks on my setup

Reply:
The project information has been added with https://github.com/denysdovhan/spaceship-prompt/pull/472/commits/5839963f8ee46472fb569cb028ff5cdad4e90944

![Screenshot 2020-05-22 at 20 09 03](https://user-images.githubusercontent.com/795289/82696873-381a4900-9c68-11ea-90eb-a4a35a659645.png)

Reply:
Resolved as stated in Resolved as stated in https://github.com/denysdovhan/spaceship-prompt/pull/472#issuecomment-632841994